telegram/ext/filters.py

```diff
@@ -1525,6 +1531,9 @@ class Regex(MessageFilter):
         With a :attr:`telegram.Message.text` of `x`, will only ever return the matches for the
         first filter, since the second one is never evaluated.
 
+    .. seealso:: `Types of Handlers <https://github.com/\
+        python-telegram-bot/python-telegram-bot/wiki/Types-of-Handlers>`_
```

Comment:
not needed imo

Reply:
I guess I'll have to wait for comment from @Bibo-Joshi here because in [this comment](https://github.com/python-telegram-bot/python-telegram-bot/pull/3306#discussion_r1004886542) there was a suggestion that I should remove links to 2 examples (which I did) but not to "Types of Handlers" wiki (so I left it there)

Reply:
I'm fine with either way. You were very thorough on adding referencings, so we're bound to have a few references that may not be necessary (depending on who's judging :D), but a few more is surely better than less :)

Reply:
ok eh let's keep it